### PR TITLE
enhancement[chroma]: Added a function `from_existing_collection`.

### PR DESCRIPTION
**Pull request by spike-spiegel-21**
Resolves this: #23797

### CODE REVIEW

**Comment by RafaelXokito** [comment]
I have reviewed the `from_existing_collection` method, and the method only calls the class constructor with the same parameters. This raises a concern about potential redundancy. Users can achieve the same result by directly calling the Chroma constructor with the appropriate arguments (As shown in the issue you raised).

File: libs/partners/chroma/tests/integration_tests/test_vectorstores.py [hunk 1]
```diff
@@ -528,3 +528,30 @@ def test_reset_collection(client: chromadb.ClientAPI) -> None:
     assert vectorstore._collection.count() == 0
     # Clean up
     vectorstore.delete_collection()
+
+
+def test_from_existing_collection() -> None:
```
Comment (RafaelXokito) [inline libs/partners/chroma/tests/integration_tests/test_vectorstores.py:533]: Duplicated of `test_chroma_with_persistence` test

**Comment by spike-spiegel-21** [comment]
> I have reviewed the `from_existing_collection` method, and the method only calls the class constructor with the same parameters. This raises a concern about potential redundancy. Users can achieve the same result by directly calling the Chroma constructor with the appropriate arguments (As shown in the issue you raised).

Makes sense @RafaelXokito

**Comment by spike-spiegel-21** [comment]
I am closing this PR. Thank you @RafaelXokito for your feedback.

**Pull request closed without merging**